doc/data/messages/i/import-self/bad.py

```diff
@@ -0,0 +1,4 @@
+cat > foo.py <<EOF

```

Comment:
This one is also tricky, maybe it would work by importing ``bad`` ?

Reply:
Thanks for the suggestion! It seems that the test error is resolved by importing `bad`, since the file is named `bad.py`.